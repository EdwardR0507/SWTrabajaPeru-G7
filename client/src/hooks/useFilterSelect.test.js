/*Importamos las librerias principales*/
import React from "react";
import '@testing-library/jest-dom/extend-expect';
import { shallow } from "enzyme";
import UseFilterSelect from "./useFilterSelect";
/*Renderizado del componente UseFilterSelect*/
describe("UseFilterSelect", () => {
  it('render "UseFilterSelect"', () => {
    const wrapper = shallow(<UseFilterSelect />);
    expect(wrapper).toMatchSnapshot()
  });
});
