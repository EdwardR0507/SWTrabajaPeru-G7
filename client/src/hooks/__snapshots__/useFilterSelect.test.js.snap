// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UseFilterSelect render "UseFilterSelect" 1`] = `ShallowWrapper {}`;
